alacritty.yml

```diff
@@ -253,6 +253,10 @@ mouse:
 
 selection:
   semantic_escape_chars: ",│`|:\"' ()[]{}<>"
+  # When set to true, selection string will be copy to primary clipborad.
+  # Default value is false, the selection string will be copy the selection
+  # clipborad.
+  save_to_clipboard: false
```

Comment:
Please also add this to `alacritty_macos.yml`. Should the default be true on that platform?